### ISSUE TITLE
[BUG]  undefined: syscall.SIGTSTP

### ISSUE THREAD

**Issue by thedolphincampaign** [hibiken/asynq#129]
Hi,
I am getting an error below when running example code on Windows. I enabled redis server
```

> D:\go-work\pkg\mod\github.com\hibiken\asynq@v0.7.0\background.go:257:55: undefined: syscall.SIGTSTP
> D:\go-work\pkg\mod\github.com\hibiken\asynq@v0.7.0\background.go:260:13: undefined: syscall.SIGTSTP
```

**Comment by hibiken** [comment]
@thedolphincampaign Thank you for reporting this issue.

Looks like [windows does not have SIGTSTP signal](https://github.com/golang/go/issues/32181). 
I'm going to resolve this issue ASAP so that people can use asynq package on windows.

**Comment by hibiken** [comment]
@thedolphincampaign It should be fixed in v0.7.1. Please let me know if you still encounter any issues.

**Comment by thedolphincampaign** [comment]
@hibiken  It worked. Thank you for fixing it :)
